js/header-resizer.js

```diff
@@ -0,0 +1,63 @@
+/* global $ $$ dom */// dom.js
+/* global debounce */// toolbox.js
+/* global prefs */
+'use strict';
+
+(() => {
+  let curW, offset, active;
+  prefs.subscribe(dom.HWprefId, (key, val) => {
+    if (!active && val !== curW) {
+      getCurWidth();
```

Comment:
It seems that we can remove this line.

When `active`, this line won't be called.

When `!active`, this is only used to decide whether or not to update CodeMirror widgets. The `delta` check at L55 seems also partly overlap with L9.

Reply:
Yeah.